include/mesoeast.php

```
@@ -1,15 +1,15 @@
 <?php
 require_once "../../../include/jpgraph/jpgraph.php";
 require_once "../../../include/jpgraph/jpgraph_led.php";
-
-function dwpf($tmpf, $relh)
-{
-    $tmpk = 273.15 + (5.00 / 9.00 * ($tmpf - 32.00));
-    $dwpk = $tmpk / (1 + 0.000425 * $tmpk * - (log10($relh / 100)));
-    return round(($dwpk - 273.15) * 9.00 / 5.00 + 32, 2);
-}
+require_once "../../../include/mlib.php";
```

PHPStan (2.1.55)

[error] Path in require_once() "../../../include/mlib.php" is not a file or it does not exist.


(requireOnce.fileNotFound)